scripts/core/editor3/store/index.ts

```diff
@@ -193,15 +195,20 @@ export default function createEditorStore(
     if (spellcheck != null) {
         // after we have the dictionary, force update the editor to highlight typos
         spellcheck.getDict().finally(() => store.dispatch(forceUpdate()));
-
         spellcheck.getAbbreviationsDict().then((abbreviations) => {
             store.dispatch(setAbbreviations(abbreviations || {}));
         });
     }
 
+    editor3Stores.push(store);
```

Comment:
when does it remove stores? it seems like it would be pushing all the time, that might use quite some memory eventually